app/assets/javascripts/views/index_table_view.js.coffee

```diff
@@ -115,7 +133,7 @@ class ELMO.Views.IndexTableView extends ELMO.Views.ApplicationView
   # event handler for when a checkbox is clicked
   checkbox_changed: (event) ->
     # unset the select all field if a checkbox is changed in any way
-    @select_all_field.val('')
```

Comment:
Let's spend a bit of time pairing on this file. It's kind of a mess. Now that we have good test coverage we should refactor it a bit.